std_msgs/Header                 header
float64                         lat
float64                         lon
string                          uri
int64                           port
string                          mountpoint